lib/features/authentication/domain/entities/authenticated_user.dart

```diff
@@ -3,12 +3,19 @@ import 'package:equatable/equatable.dart';
 class AuthenticatedUser extends Equatable {
   final String email;
   final String token;
+  final String encodedPasscode;
+  final DateTime? lastLogin;
```

Comment:
Why would this value be nullable?

Reply:
Because the user may not have been logged in before. Users can also sign out